### ISSUE TITLE
If i'm intending on using tensor flow? 

### ISSUE THREAD

**Issue by AliaMYH** [MarcBS/keras#37]
I want to convert my caffe network to keras, but this fork only supports this for theano. I'm intending on using the tensorflow backend for my project, and was wondering how i could do that? What would i have to modify in the created models? 

I'm not entirely sure what the difference between the two backends are besides the channel ordering. 

@MarcBS

**Comment by MarcBS** [comment]
If you are simply interested in converting models and using the normal layers available in Keras then there should be no problem. You should be able to use either TF or TH.

The thing is that we can only assure that the additional multimodal layers that we have developed are working on TH. Not all of them are guaranteed to work on TF.

**Comment by AliaMYH** [comment]
The multimodal layers being the LSTMs etc? If my model only has the typical conv, maxpooling and dropout etc it should be fine ?

My goal is to convert the models, and then train them, or use them to make predictions

**Comment by lvapeab** [comment]
Closing this, as we now fully support TF.
